plutus-pab-client/default.nix

```diff
@@ -27,7 +27,8 @@ let
   migrate = pkgs.writeShellScriptBin "plutus-pab-migrate" ''
     # There might be local modifications so only copy when missing
     ! test -f ./plutus-pab.yaml && cp ../plutus-pab/plutus-pab.yaml.sample plutus-pab.yaml
-    $(nix-build ../default.nix --quiet --no-build-output -A plutus-pab.server-setup-invoker)/bin/plutus-pab-setup migrate
+    # TODO: Change the migrate to read the config file instead and get the db path from there
+    $(nix-build ../default.nix --quiet --no-build-output -A plutus-pab.server-setup-invoker)/bin/plutus-pab-setup migrate $(yq -r '.dbConfig.dbConfigFile' plutus-pab.yaml)
```

Comment:
Relevant to create an issue where we modify the migrate command in the PAB to read the `.dbConfig.dbConfigFile` from the YAML config file instead of reading the YAML file with `yq`?

Reply:
Yeah, we initially did it like that because of the VM tests. But I agree it would be easier to just get it from the config. I'll add a ticket for that.

Reply:
https://jira.iohk.io/browse/SCP-2600